src/config.ts

```diff
@@ -31,6 +31,23 @@ export default function getConfig<T extends ConfigValue>(key: string, defaultVal
 	throw new Error(error) || new Error("Config key " + key + " not found")
 }
 
+function getEnvVar(key: string): ConfigValue | undefined {
+	const value = process.env[key] ?? process.env[key.replaceAll(".", "_")]
+	if (!value) return
+
+	return parseStringValue(value)
+}
+
+function parseStringValue(value: string): ConfigValue {
+	// boolean
+	if (value == "true") return true
+	else if (value == "false") return false
```

Comment:
if(Boolean(value)) return !!value;
would something like that work?

Reply:
no, any string that isn't empty ("") is a truthy value
so Boolean(value) would always be true and it'll return true (Boolean(x) does the same as !!x)
I can
```suggestion
	if (["true", "false"].includes(value)) return !!value
```
tho
but it doesn't really become any more readable that way 🤷‍♀️

Reply:
ok let's keep it